Engine/UserInput.cs

```diff
@@ -175,6 +216,8 @@ void IUserInputInterface.RequestUserInput(object dataObject, TimeSpan timeout, b
                 if (display is DisplayAttribute attr && attr.IsDefaultAttribute() == false)
                     title = display.Name;
                 
+                // flush and make sure that there is no new log messages coming in (0.1s span).
+                Log.Flush(TimeSpan.FromSeconds(0.1));    
```

Comment:
I'm not sure I understand why a normal flush is not enough. I'm guessing we do this to avoid getting update info printed while we are waiting for user input. It looks to me like that would be covered by the userInputMutex (and a regular flush).

Reply:
Done! I don't think we actually needed that.